Library/Homebrew/cmd/update-reset.sh

```diff
@@ -42,7 +42,9 @@ homebrew-update-reset() {
     echo
 
     ohai "Resetting $DIR..."
-    git checkout --force -B master origin/master
+    head="$(git symbolic-ref refs/remotes/origin/HEAD)"
```

Comment:
`fatal: ref refs/remotes/origin/HEAD is not a symbolic ref` on my checkout.

Reply:
How bizarre. The ref `refs/remotes/origin/HEAD` is fetched on line 41.
```bash
git remote set-head origin --auto >/dev/null
```
Which repository gave this error? Is it reproducible? Does the file `.git/refs/remotes/origin/HEAD` exist, and if so what's its contents? What's the output of `git remote set-head origin --auto`?

Reply:
Sorry, I missed the `set-head` above. This was on my local checkout. Once running the command above: this works.